controller/api/apis.go

```diff
@@ -3625,3 +3632,30 @@ type REST_SigstoreRootOfTrustCollection struct {
 type REST_SigstoreVerifierCollection struct {
 	Verifiers []REST_SigstoreVerifier `json:"verifiers"`
 }
+
+type REST_RemoteExportRepository_POST struct {
+	Alias                            string `json:"alias"`
+	RepositoryOwnerUsername          string `json:"repository_owner_username"`
+	RepositoryName                   string `json:"repository_name"`
+	RepositoryBranchName             string `json:"repository_branch_name"`
+	PersonalAccessToken              string `json:"personal_access_token"`
+	PersonalAccessTokenCommitterName string `json:"personal_access_token_committer_name"`
+	PersonalAccessTokenEmail         string `json:"personal_access_token_email"`
+}
+
+type REST_RemoteExportRepository_GET struct {
```

Comment:
"PersonalAccessToken" field is not in this type. How does user know it's empty or configured?

Reply:
1. I wonder if we should expose the personal access token after it has been initialized, perhaps another field is necessary. We can discuss this.
2. Good point, I have added a nickname/alias field to the remote repository objects.